### ISSUE TITLE
`Lint/UnusedMethodArgument` should ignore keyword arguments

### ISSUE THREAD

**Issue by johnnyshields** [rubocop/rubocop#11811]
`Lint/UnusedMethodArgument` will be triggered on the following:

```ruby
def my_method(my_kwarg: false)
  # do nothing
end

# triggers Lint/UnusedMethodArgument: Unused method argument - my_kwarg
```

Since the name of keyword arguments is important, I think `Lint/UnusedMethodArgument` should leave kwargs alone.

Consider code like the following:

```ruby
class MyClass1
  def my_method(my_kwarg: false)
    puts my_kwarg
  end
end

class MyClass2
  def my_method(my_kwarg: false)
    # do nothing
  end
end

(some_condition ? MyClass1 : MyClass2).my_method(my_kwarg: :foo)
```

This will break if I rename `my_kwarg` to `_my_kwarg` in `MyClass2`

**Comment by koic** [comment]
`AllowUnusedKeywordArguments: true` seems to solve it:
https://docs.rubocop.org/rubocop/1.50/cops_lint.html#examples-121

**Comment by johnnyshields** [comment]
Sorry missed that. Thanks!
